h/static/scripts/widget-controller.js

```diff
@@ -297,6 +301,17 @@ module.exports = function WidgetController(
     }
   }
 
+  $rootScope.$on('sidebarOpened', function () {
+    sidebarOpen = true;
+    streamer.connect();
+  });
+
+  $rootScope.$on(events.USER_CHANGED, function () {
+    if (sidebarOpen) {
```

Comment:
I think this was faulty logic from my original patch. The transition here can either be from logged out -> logged in, in which case the streamer should be connected, or logged in -> logged out, in which case the streamer connection also needs to be torn down and recreated.

So indeed, it looks like we don't need this flag any more.